_posts/2017-01-13-create-android-background-service.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Creating background services in Android
+ghavatar: 2993240
+ghname: jarlefosen
+jobtitle: Platform Engineer
+tags: [android, sdk]
+---
+
+## Introduction
+
+<div class="message">
+    How do you create a background service in Android that lives its own life?
+    Could you also communicate with it?
+</div>
+
+## Android background service?
+
+> I want some stuff to happen in the background of my application, even if my application is closed!
+> You see, I really want some function to be invoked every second or so which tells me the current time.
+> And if the application is open, it should be able to display this time.
```

Comment:
>  And if the application is open, it should be able to display this time.

Something is off with that sentence. Rephrase to something like "And if the application is open the current time should be displayed"